edge-util/src/Microsoft.Azure.Devices.Edge.Util/PeriodicTask.cs

```diff
@@ -85,9 +89,22 @@ async Task DoWork()
                 {
                     try
                     {
-                        this.logger.LogInformation($"Starting periodic operation {this.operationName}...");
+                        // void Log(string message) => this.verbose ? this.logger.LogInformation(message) : this.logger.LogDebug(message);
+                        void Log(string message)
+                        {
+                            if (this.verbose)
+                            {
+                                this.logger.LogInformation(message);
+                            }
+                            else
+                            {
+                                this.logger.LogDebug(message);
+                            }
```

Comment:
What is the difference between these methods?

Reply:
debug is more verbose than information. So if log output is set to information and below, debug won't be output.